src/tezos/tezosPlugin.js

```diff
@@ -56,7 +65,7 @@ export class TezosPlugin extends CurrencyPlugin {
         privateKey: keys.sk
       }
     } catch (e) {
-      throw new Error('Invalid input')
+      throw new Error(e)
```

Comment:
This code is wrong. `throw new Error(e)` doesn't even make sense - `e` is an `Error` object, not a string. Just get rid of the entire try/catch block.